add polynomial-decay-helpers (#16)
climate-resource/gradient-aware-harmonisation: changelog/17.fix.md

> + fix computation of cosine decay derivative (we forgot a constant value in the formula)

Reviewer: ```suggestion
+ Fixed computation of cosine decay derivative (a constant factor was missing)
```

(nwtkia)
